app/components/Sidebar/index.tsx

```diff
@@ -210,7 +210,7 @@ const Sidebar = (): JSX.Element => {
           {drawer}
         </Drawer>
       </Hidden>
-      <Hidden xsDown>
+      <Hidden smDown>
```

Comment:
it doesn't look like xs v4 is diff from xs v5? why was this change here

Reply:
Not sure, I'll look into it

Reply:
https://mui.com/guides/migration-v4/#theme there was changes, but I don't know if they necessarily apply here. `xsDown` seems to mean less than 0px, so I think this change is correct. I tested it and `smDown` works for the behaviour I want.